sql/moz-fx-data-shared-prod/telemetry/nondesktop_clients_last_seen_v1/view.sql

```diff
@@ -45,6 +45,8 @@ unioned AS (
     days_since_seen,
     days_created_profile_bits,
     days_since_created_profile,
+    first_seen_date,
+    is_new_profile,
```

Comment:
These fields are both covered by the `SELECT *` from core_clients_daily, but there's a wrinkle as far as how we want to count new profiles from last_seen. If we do `COUNTIF(is_new_profile)`, we'll overcount since a client that appears on a single day will continue to appear in clients_last_seen with `is_new_profile=True` carried over from the original day of observation. We'd need to do `COUNTIF(first_seen_date = submission_date) AS new_profiles` like we do for desktop.

I've updated this PR to include `is_new_profile` in the `REPLACE` and make sure it's null for days on which the client isn't observed. But this differs from normal clients_last_seen semantics for dimensions, and requires and update to the baseline tables as well.

But perhaos we should not include `is_new_profile` at all in clients_last_seen to avoid confusion.

Reply:
This could use @irrationalagent's feedback as well.

Reply:
Ah right, I forgot about the `select *` behavior of the last seen table. It should only be set when the first_seen_date is equal to the submission date, it doesn't sound like it should be overcounting if the semantics are the same as the glean tables.

Reply:
> It should only be set when the first_seen_date is equal to the submission date, it doesn't sound like it should be overcounting if the semantics are the same as the glean tables.

The backfill that we did for the Glean tables is legit. That would have only set `is_new_profile` when the clients_last_seen record had a matching submission_date.

But the incrementals since the backfill are not correct. See https://sql.telemetry.mozilla.org/queries/79491/source

That query shows that starting on 2021-04-11 (first day after backfill of first_seen_date), we start getting non-zero counts for clients that have `is_new_profile` set, but `submission_date != first_seen_date`. I think we should remove this field.

Reply:
Specifically, I am going to remove `is_new_profile` via an EXCEPT clause in this PR.

I think we should file a follow-up to remove the field from baseline_clients_last_seen, or at least to exclude it at the view level.